src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackProcessorImpl.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd;
+
+import io.vavr.Value;
+import io.vavr.control.Validation;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import java.util.List;
+
+import static io.vavr.control.Validation.invalid;
+import static io.vavr.control.Validation.valid;
+import static java.lang.String.format;
+import static java.util.Collections.emptyList;
+import static uk.gov.hmcts.reform.ccd.client.model.CallbackTypes.ABOUT_TO_SUBMIT;
+
+@Service
+public class CallbackProcessorImpl implements CallbackProcessor {
+
+    @Override
+    public List<String> process(String type, String eventId, CaseDetails caseDetails) {
+        return Validation
+            .combine(
+                isAttachEvent(type),
+                isAboutToStart(eventId)
+            )
+            .ap((aType, anEventId) -> attach(caseDetails))
+            .getOrElseGet(Value::toJavaList);
+    }
+
+    private List<String> attach(CaseDetails caseDetails) {
+        return emptyList();
+    }
```

Comment:
why empty list? is this a temporary solution? If so, please leave a comment in code.

Reply:
Empty list means no Errors .. therefore success :D